docs/eventing/experimental-features.md

```diff
@@ -103,6 +103,101 @@ affecting existing resources.
     At the moment this feature is implemented only for
     `Subscription.Spec.Subscriber.Ref` and `Subscription.Spec.Channel`.
 
+### DeliverySpec.RetryAfterMax field
+
+**Flag name**: `delivery-retryafter`
+
+**Stage**: Alpha, disabled by default
+
+**Tracking issue**: [#5811](https://github.com/knative/eventing/issues/5811)
+
+**Persona**: Developer
+
+When using the `delivery` spec to configure event delivery parameters, you can
+use the `retryAfterMax` field to specify how
+HTTP [Retry-After](https://datatracker.ietf.org/doc/html/rfc7231#section-7.1.3)
+headers are handled when calculating backoff times for retrying **429** and
+**503** responses. You can specify a `delivery` spec for Channels,
+Subscriptions, Brokers, Triggers, and any other resource spec that accepts the
+`delivery` field.
+
+The `retryAfterMax` field will only take effect if the `delivery` spec is
+configured to perform retries, and will only pertain to retry attempts on
+**429** and **503** response codes. The field provides an override to prevent
+large **Retry-After** durations from impacting throughput, and must be specified
+using the [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Times) format. The
+largest of the normal backoff duration and the Retry-After header value will be
+used for the subsequent retry attempt. Specifying a "zero" value of "PT0S" will
+effectively disable **Retry-After** support.
+
+Prior to this experimental feature, Knative Eventing implementations have not
+supported **Retry-After** headers, and this is an attempt to provide a path
+for standardizing that support.  To begin the feature will be **opt-in**, but
+the final state will be **opt-out** as follows...
+
+<table>
```

Comment:
Can this table be in markdown?
Ref: https://github.github.com/gfm/#tables-extension-

Reply:
definitely - have reworked it - thanks for reviewing!